### DISCUSSION TITLE
Thematic update: Camping

### DISCUSSION THREAD

**Discussion by Outlet2048** [organicmaps/organicmaps#3189] [Ideas]
I was thinking that maybe it would be cool if Organic Maps (OM) released thematic updates that specialized in improving a particular part of the application.

Mainly it's about displaying more information for related POIs, as well as rendering more POIs that can help the target audience
Since OM is currently developed more for the outdoor target group, I thought that one could start with camping as an example

For this I have compiled a list of suggestions:

**Rendering new stuff:**
-----

- Render the outline of a camping place
 #3190

- Render camp pitches: 
 #1660

- render amenity=reception
#3191

- render fire pits
#3580

- Render more elements from the comment here: 
https://github.com/organicmaps/organicmaps/issues/1577#issuecomment-979979451

Displaying more POI information
------

Many POIs are already being rendered but lack certain information that is very useful to travelers 

- Display whether or not toilets and showers are free to use, if not displaying the cost. (not just for campers. All users would profit from this)
#3192

- show more information for the camping place itself, such as:
    [Presence and specifications of power supplies ](https://wiki.openstreetmap.org/wiki/Key:power_supply)
    [Whether or not internet is available and at what cost ](https://wiki.openstreetmap.org/wiki/Key:internet_access)
    [Whether or not dogs are allowed on the premise, and whether they have to be leashed]( https://wiki.openstreetmap.org/wiki/Key:dog)
  
 - The following three tags help giving the user more information when the POIs itself arent mapped seperately. Therefore it is still useful to show these details when the users clicks on a camping place

    [Whether or not drinking water is available](https://wiki.openstreetmap.org/wiki/Key:drinking_water)  
    [Whether the camping place has a washing mashine]( https://wiki.openstreetmap.org/wiki/Key:washing_machine)
    [Whether the camping place has a dryer ](https://wiki.openstreetmap.org/wiki/Key:dryer)

- Display a description when clicking on a POI (also useful for all users)
https://github.com/organicmaps/organicmaps/issues/1888
https://github.com/organicmaps/organicmaps/issues/1233
https://github.com/organicmaps/organicmaps/pull/1929
https://github.com/organicmaps/organicmaps/issues/1953
The PR makes it seem like this has already been implemented but I haven't seen the description field being shown in the app so far, so I assume it hasnt been completed yet

More information and suggested tags can be found [here](https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcamp_site)
This [page ](https://wiki.openstreetmap.org/wiki/Proposed_features/Extend_camp_site)contains a list of example taggings that can be used on camping places and could also be displayed 

[Example camping place](https://www.openstreetmap.org/way/76782881) 

🍃 Please feel free to comment with suggestions and I'll try my best to update this post ASAP

**Comment by biodranik** [comment]
Thanks for the excellent summary, these are all great improvements that I was missing while camping and traveling :) Aren't dump stations already supported? Did you check the "RV" category in the search?
It would be better to create an issue with this info and link it with other issues from there.
CC @pastk

**Reply by Outlet2048** [reply]
Thanks for the quick feedback
I went ahead and updated the post with the issues I've created

Seems like RV dumping stations are being shown and the issue I had was due to tagging in OSM Data.

Please feel free to suggest more changes based on your camping / travel experience and I'll try my best to keep this list updated :)

**Reply by biodranik** [reply]
All mentioned issues should keep in mind an easy-to-use map editing functionality in OM. Then users will generate better content even from offline remote regions.

**Reply by Outlet2048** [reply]
> All mentioned issues should keep in mind an easy-to-use map editing functionality in OM. Then users will generate better content even from offline remote regions.


Most of those tags could be mapped by the user with simple yes or no drop down fields.

(Examples: Presence of features, such as dryers or washing mashines, availability of internert, and whether or not something is free to use)
In some cases we could take a look at how StreetComplete handles these questions and what options the users is presented with.


Does every tag that is displayed require the option to be edited by the user?

Althoug useful, this seems to be limiting the amount of information that can be displayed to the user. 
For example: Showing the user the "charge" tag when viewing toilets or showers can be very useful. But letting the user edit this data requires the users to know the correct syntax for the tag in question. I believe most users would get it wrong since the syntax defined in the wiki is very specific

So how about displaying such information regardless but not giving the user the option to edit it? 
Or creating a note upon saving the edit, instead of editing the actual tag
Another option would be a small disclaimer at the bottom of every POI that explains that data is sourced from OpenStreetMap with the option to add it there, just how Qwant Maps does it:

![grafik](https://user-images.githubusercontent.com/72819077/185622185-b6192e25-8265-4aae-8924-cf938c8fba30.png)

**Reply by TheAdventurer64** [reply]
> > All mentioned issues should keep in mind an easy-to-use map editing functionality in OM. Then users will generate better content even from offline remote regions.
> 
> Most of those tags could be mapped by the user with simple yes or no drop down fields.
> 
> (Examples: Presence of features, such as dryers or washing mashines, availability of internert, and whether or not something is free to use) In some cases we could take a look at how StreetComplete handles these questions and what options the users is presented with.
> 
> Does every tag that is displayed require the option to be edited by the user?
> 
> Althoug useful, this seems to be limiting the amount of information that can be displayed to the user. For example: Showing the user the "charge" tag when viewing toilets or showers can be very useful. But letting the user edit this data requires the users to know the correct syntax for the tag in question. I believe most users would get it wrong since the syntax defined in the wiki is very specific
> 
> So how about displaying such information regardless but not giving the user the option to edit it? Or creating a note upon saving the edit, instead of editing the actual tag Another option would be a small disclaimer at the bottom of every POI that explains that data is sourced from OpenStreetMap with the option to add it there, just how Qwant Maps does it:
> 
> ![grafik](https://user-images.githubusercontent.com/72819077/185622185-b6192e25-8265-4aae-8924-cf938c8fba30.png)
See also #2119 and 2254

**Comment by hb0nd** [comment]
this would be good for this too: https://github.com/organicmaps/organicmaps/discussions/3580

**Reply by Outlet2048** [reply]
added to list
